### PR TITLE
Openvpn up/down scripts support

### PR DESCRIPTION
**Pull request by kohoutovice**
Hello,
I added support for openvpn up/down scripts. So if you'd like it you can merge it.

### CODE REVIEW

File: templates/server.conf.j2 [hunk 1]
```diff
@@ -118,3 +118,12 @@ verify-client-cert {{ ldap.verify_client_cert }}
...
 {% endif %}
 {% endif %}
+
+script-security {{ openvpn_script_security }}
+
+{% if openvpn_script_up is defined %}
+up {{openvpn_base_dir}}/up.sh
+{% endif %}
+{% if openvpn_script_down is defined %}
+up {{openvpn_base_dir}}/down.sh
```
Comment (kyl191) [inline templates/server.conf.j2:128]: Should this be `down`?

**Review by kyl191 — CHANGES_REQUESTED** [review]
Just a weird thing on the `down/up` in server.conf.j2

**Comment by kohoutovice** [comment]
Yeah, there was a typo mistake. Sorry for that. Now I've made triple check of my changes and tested it in my local enviroment. 

In addition I added the option for client dis/connect script.

**Comment by kyl191** [comment]
lgtm. Centos failures are an upstream Ansible problem: https://github.com/ansible/ansible/pull/73689 

Merging.

**Pull request merged**